Server/sprawl_calculator/wup_calculator/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+import uuid
+
+# Create your models here.
+
+class Model(models.Model):
+	id = models.UUIDField() #Id
+	field1 = models.CharField() #Character
+	field2 = models.DateTimeField() #Time
+	status = models.IntegerField() #Status
```

Comment:
See the [documentation](https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L315) of the status, it is not a integer, if you want to make it a integer, use a lookup / choice field or something similar.